conda_smithy/configure_feedstock.py

```diff
@@ -390,7 +390,7 @@ def main(forge_file_directory):
               'circle': {},
               'appveyor': {},
               'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]},
-              'github': {'user_or_org': 'conda-forge', 'repo_name': ''},
+              'github': {'user_or_org': 'conda-forge', 'repo_name': '', 'branch_name': 'master'},
```

Comment:
So knowing this info may not be redundant. Think whitelisting release branches ( https://github.com/conda-forge/conda-forge.github.io/issues/87 ). Will flesh out more in issue ( https://github.com/conda-forge/conda-smithy/issues/318 ).

FYI the emojis are suppose to mean Australia. 😄